src/core/NotePlayHandle.cpp

```diff
@@ -252,8 +252,17 @@ void NotePlayHandle::play( sampleFrame * _working_buffer )
 	if( m_released && (!instrumentTrack()->isSustainPedalPressed() ||
 		m_releaseStarted) )
 	{
-		m_releaseStarted = true;
+		if (m_releaseStarted == false)
```

Comment:
Don't mix styles. This would be comparably to the rest of your changes in this PR and surrounding code.

```cpp
if( m_releaseStarted == false )
```